command/core.py

```diff
@@ -9,13 +10,14 @@
 class Core:
     """Encapsulate methods for handling events."""
 
-    def __init__(self, db_facade, bot, gh_interface):
+    def __init__(self, db_facade, bot, gh_interface, token_config):
         """Initialize the dictionary of command handlers."""
         self.__commands = {}
         self.__facade = db_facade
         self.__bot = bot
         self.__github = gh_interface
         self.__commands["user"] = UserCommand(self.__facade, self.__github)
+        self.__commands["token"] = TokenCommand(self.__facade, token_config)
```

Comment:
Is it time to finally start looking into meta-classes? :p

Reply:
You don't have a task anymore, right? You could definitely start looking into that now, if you'd like.

Reply:
I thought I was supposed to do team commands as well? But yea